frontend/src/app/home/AudioInput.tsx

```
@@ -0,0 +1,185 @@
+    container: containerRef,
+    height: 30,
+    barHeight: 10,
+    barWidth: 1,
+    waveColor: getColor(reviewType, 'wave'),
+    progressColor: getColor(reviewType, 'progress'),
+    url: blobUrl,
+    // plugins: useMemo(() => [Timeline.create()], []),
+  })
+
+  const onPlayPause = useCallback(() => {
+    wavesurfer && wavesurfer.playPause()
```

Biome

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)